plugins/woocommerce/client/legacy/js/admin/meta-boxes-product.js

```diff
@@ -902,6 +936,34 @@ jQuery( function ( $ ) {
 		}
 	);
 
+	// Go to variations tab when clicking on link in the general tab message
+	$( document.body ).on(
+		'click',
+		'#general_product_data .woocommerce-message a[href="#variable_product_options"]',
+		function () {
+			$(
+				'#woocommerce-product-data .variations_tab a[href="#variable_product_options"]'
+			).trigger( 'click' );
+			return false;
+		}
+	);
+
+	// Go to linked products tab when clicking on link in the general tab message
+	$( document.body ).on(
```

Comment:
Same here, and I would also add that modification to [this line](https://github.com/woocommerce/woocommerce/pull/38531/files#diff-80a64e4128666df5c4ab90c1f3f859e14fee43f666f455f823d80d47ed0df506R928).